### PR TITLE
fixing the care and support update form

### PR DESCRIPTION
**Pull request by Mathew77**
fixing the care and support update form

### CODE REVIEW

File: src/main/java/org/lamisplus/modules/hiv/repositories/HivEnrollmentRepository.java
Comment (lilkersy) [inline src/main/java/org/lamisplus/modules/hiv/repositories/HivEnrollmentRepository.java:1]: Was the group by statement in the previous query giving an issue? I see that the whole query was changed to reflect the correct parameters for the "ilike"?

File: src/main/java/org/lamisplus/modules/hiv/service/HivPatientService.java
Comment (lilkersy) [inline src/main/java/org/lamisplus/modules/hiv/service/HivPatientService.java:1]: What is the difference between if (!((searchValue == null) || (searchValue.equals("*")))) and if ((searchValue != null) || (!searchValue.equals("*"))). If the former seems to work. I would be curious to know why though.

File: src/main/webapp/jsx/components/ChronicCare/Tb.js
Comment (lilkersy) [inline src/main/webapp/jsx/components/ChronicCare/Tb.js:1]: These two fields           
1. ["status"]: "Currently on TB Treatment", (old) and 
2. ["status"]: "Currently on TB treatment", (new). If this appears on the UI then it I presume it would appear on the RADET as well. Considering how data are counted on the RADET, inconsistent namings might cause an issue. Can we standardize these namings (eg. using Ii8n)?

File: src/main/webapp/jsx/components/Consultation/Home.js
Comment (lilkersy) [inline src/main/webapp/jsx/components/Consultation/Home.js:1]: This is a really cool spotting and implementation. However, is there also a consideration for weight, somewhere in the code when valuating for children above say 30kg?

File: src/main/webapp/jsx/components/History/RecentHistory.js
Comment (lilkersy) [inline src/main/webapp/jsx/components/History/RecentHistory.js:1]: const [vitaLoad, setViralLoad] = useState([]); Is this mean to be viralLoad instead of VitaLoad?

File: src/main/webapp/jsx/components/Otz/ServiceForm.js
Comment (lilkersy) [inline src/main/webapp/jsx/components/Otz/ServiceForm.js:1]: Does setting this facilityId: null, (line 97) have any effect?

File: src/main/webapp/jsx/components/TrackingForm/Tracking.js
Comment (lilkersy) [inline src/main/webapp/jsx/components/TrackingForm/Tracking.js:1]: What is the difference between this "setSaving" and the "setSavings" in the OtzServiceForm.js? Is it just a nomenclature definition thing?

**Review by lilkersy — CHANGES_REQUESTED** [review]
I have taken look at the code and asked a couple of questions. Please review.

File: src/main/java/org/lamisplus/modules/hiv/repositories/HivEnrollmentRepository.java
Comment (Mathew77) [reply src/main/java/org/lamisplus/modules/hiv/repositories/HivEnrollmentRepository.java:1]: It's not but @dikumheartland  said for othername that are migrated and empty are not pulling. 
@dikumheartland can you please provide more clearity.

File: src/main/java/org/lamisplus/modules/hiv/service/HivPatientService.java
Comment (Mathew77) [reply src/main/java/org/lamisplus/modules/hiv/service/HivPatientService.java:1]: It's not but @dikumheartland said for othername that are migrated and empty are not pulling.
@dikumheartland can you please provide more clarity.

File: src/main/webapp/jsx/components/ChronicCare/Tb.js
Comment (Mathew77) [reply src/main/webapp/jsx/components/ChronicCare/Tb.js:1]: Yes it will appear, because it is just the display name that was change which RADET is using the application_codeset code to get the value.

File: src/main/webapp/jsx/components/Consultation/Home.js
Comment (Mathew77) [reply src/main/webapp/jsx/components/Consultation/Home.js:1]: Please can you provide more context to this. Because I can't see that in the home.js

File: src/main/webapp/jsx/components/History/RecentHistory.js
Comment (Mathew77) [reply src/main/webapp/jsx/components/History/RecentHistory.js:1]: Yes, it should be ViralLoad.

Comment (Mathew77) [reply src/main/webapp/jsx/components/History/RecentHistory.js:1]: Fixed

**Comment by Mathew77** [comment]
@lilkersy thank you so much for your review, corrections made where necessary.

File: src/main/webapp/jsx/components/Consultation/Home.js
Comment (lilkersy) [reply src/main/webapp/jsx/components/Consultation/Home.js:1]: Yea, I think there were previous implementations especially in the LAMIS3 where weight is also considered for peads, or are we letting go of that condition?

File: src/main/webapp/jsx/components/ChronicCare/Tb.js
Comment (lilkersy) [reply src/main/webapp/jsx/components/ChronicCare/Tb.js:1]: Alright, I think I understand.

File: src/main/webapp/jsx/components/Consultation/Home.js
Comment (Mathew77) [reply src/main/webapp/jsx/components/Consultation/Home.js:1]: On the national tool the only condition for children is to display MAU for vitals, I think we can discuss this with the BA COP just to be sure if that implementation is required.

File: src/main/webapp/jsx/components/Otz/ServiceForm.js
Comment (Mathew77) [reply src/main/webapp/jsx/components/Otz/ServiceForm.js:1]: No, that would be removed. Thanks

File: src/main/java/org/lamisplus/modules/hiv/repositories/HivEnrollmentRepository.java
Comment (dikumheartland) [reply src/main/java/org/lamisplus/modules/hiv/repositories/HivEnrollmentRepository.java:1]: The search query had `other_name ilike ''`. This made the result include lots of irrelevant records because a lot of clients have empty other_name. The fix was to make it like the other search params.

**Comment by Mathew77** [comment]
waiting for your approval @lilkersy, are they any pending issues that have not been addressed?

**Pull request closed without merging**
